src/ast/mod.rs

```diff
@@ -2782,12 +2783,30 @@ pub enum Statement {
         filter: Option<ShowStatementFilter>,
     },
     /// ```sql
+    /// SHOW DATABASES [LIKE 'pattern']
+    /// ```
+    ShowDatabases { pattern: Option<String> },
```

Comment:
Would it make sense to also use `ShowStatementFilter` to represent this (if there's a chance that the syntax can be extended)?

Reply:
Great idea